### PR TITLE
Add tagging taret to make file

### PR DESCRIPTION
**Pull request by TerryHowe**
Automate tagging of new release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -70,6 +70,11 @@ postsubmit-conformance:
 		--dry-run=false
 #	bash development/kops/prow.sh
 
+.PHONY: upload
```
Comment (abhay-krishna) [inline Makefile:73]:
```suggestion
.PHONY: tag
```

**Comment by abhay-krishna** [comment]
nit: typo in commit message - Add tagging `target` ...

**Comment by TerryHowe** [comment]
fixed, thanks!

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**